packages/dds/tree/src/test/shared-tree-core/sharedTreeCore.spec.ts

```diff
@@ -235,24 +292,85 @@ describe("SharedTreeCore", () => {
 		});
 	});
 
+	it("evicts trunk commits behind the minimum sequence number", () => {
```

Comment:
ensure nesting is covered and ensure Alex's test coverage is handled here as well